### DISCUSSION TITLE
Permitted command.

### DISCUSSION THREAD

**Discussion by sirtodion** [LTmadness/ValheimBetterServerConfig#27] [Q&A] [answered]
Hey and thank you for a great plugin to Valheim BepInEx. 

Is there any way to disable the permit function? I find it annoying that I have to permit players to my server when its already password protected. 
Maybe it's a default BepInEx function, I don't know but I can't find anywhere to turn it off. 
Thank you in advance. :)

**Comment by LTmadness** [comment, marked as answer]
just don't add anyone to permitted list and it won't check it then

**Reply by sirtodion** [reply]
Thank you! =) I did that just to test it. hehe
